src/slic3r/GUI/GUI_App.cpp

```diff
@@ -2754,7 +2754,7 @@ void GUI_App::init_label_colours()
     m_color_label_modified = is_dark_mode ? wxColour("#F1754E") : wxColour("#F1754E");
     m_color_label_sys      = is_dark_mode ? wxColour("#B2B3B5") : wxColour("#363636");
 
-#ifdef _WIN32
+#if defined(_WIN32) || defined(__linux__) || defined(__APPLE__)
```

Comment:
I'm just wondering if we even need this **`#if`** here, since every operating system is affected.
Or are there more OS'es than Linux, Windows & Mac? 😊 🤷‍♂️

Reply:
I suggest to let @SoftFever have a look. 😊